lib/elixir/lib/enum.ex

```diff
@@ -1445,9 +1445,9 @@ defmodule Enum do
   ## Examples
 
       iex> Enum.reduce_while(1..100, 0, fn i, acc ->
-      ...>   if i <= 3, do: {:cont, acc + i}, else: {:halt, acc}
+      ...>   if i < 3, do: {:cont, acc + i}, else: {:halt, acc}
```

Comment:
@doomspork  Well, my point was that if there wasbthe exact samd tests in the docs as in the tests we could do away without the tests. Personally I would favour to have those same tests as examples in the docs, as they will improve the documentation.